app/lib/account/backend.dart

```diff
@@ -84,6 +84,28 @@ Future<User> requireAuthenticatedUser() async {
   return _authenticatedUser;
 }
 
+/// Authenticates with bearer [token] and populates [_authenticatedUser] with
+/// it, running in a new scope.
```

Comment:
Isn't it better to explain that `requireAuthenticatedUser() ` will not fail when called in the `Zone` that `fn` is executed in.

Because that's the external interface... `_authenticatedUser` is internal to this library.

Reply:
It still fails if the user authentication failed. In such cases we don't block the requests. Maybe we should though...